api/token.ts

```diff
@@ -3,11 +3,13 @@ import { PluggyClient } from 'pluggy-sdk';
 
 const { PLUGGY_CLIENT_ID, PLUGGY_CLIENT_SECRET } = process.env;
 
-export default function (_: NowRequest, res: NowResponse) {
+export default function (req: NowRequest, res: NowResponse) {
   const pluggyClient = new PluggyClient({
     clientId: PLUGGY_CLIENT_ID,
     clientSecret: PLUGGY_CLIENT_SECRET,
   });
 
-  pluggyClient.createConnectToken().then(data => res.json(data));
+  const { itemId, options } = req.body || {}
```

Comment:
I think is not necessary the OR operator
```suggestion
  const { itemId, options } = req.body
```